src/bin/cargo/commands/config.rs

```diff
@@ -34,7 +34,7 @@ pub fn exec(config: &mut Config, args: &ArgMatches) -> CliResult {
         Some(("get", args)) => {
             let opts = cargo_config::GetOptions {
                 key: args.value_of("key"),
-                format: args.value_of("format").unwrap().parse()?,
+                format: args.value_of_t_or_exit("format"),
```

Comment:
We shouldn't be short-circuiting cargo's exiting.  We'll change expected exit codes, if nothing else.

Reply:
Thanks for the swift review. can I use `value_of_t("format")?` instead?

Reply:
Yes

Reply:
ta! will change to that